src/Sulu/Bundle/MediaBundle/Markup/MediaTag.php

```diff
@@ -144,9 +152,46 @@ function ($attributes) {
         $result = [];
         foreach ($medias as $media) {
             $media['url'] = $this->mediaManager->getUrl($media['id'], $media['name'], $media['version']);
+            $dispositionType = $this->findMediaDispositionType($media);
+
+            if ($dispositionType) {
+                $media['url'] .= '&' . $dispositionType . '=true';
+            }
+
             $result[$media['id']] = $media;
         }
 
         return $result;
     }
+
+    /**
+     * Find disposition type by media MIME type.
+     *
+     * @param array $media
+     *
+     * @return string|null
+     */
+    private function findMediaDispositionType(array $media)
+    {
+        if (
+            !isset($media['mimeType']) ||
+            !$media['mimeType'] ||
+            !isset($this->dispositionType['default']) ||
+            !$this->dispositionType['default']
+        ) {
+            return null;
+        }
+
+        foreach ($this->dispositionType as $key => $mimeTypes) {
+            if ('default' === $key || !is_array($mimeTypes) || !in_array($media['mimeType'], $mimeTypes)) {
```

Comment:
Is it possible that `$mimeTypes` is not an array? If yes I would move that to separate check and use `break` instead of `continue`, because that won't change within this loop.